utils/src/print.c

```diff
@@ -1006,6 +1002,13 @@ static int print_volume(int fd)
 
 	print_super_block(super, SCOUTFS_SUPER_BLKNO);
 
+	if (!(le64_to_cpu(super->flags) && SCOUTFS_FLAG_IS_META_BDEV)) {
```

Comment:
Ha!   I just noticed that this should also be "&", not "&&" :)

Reply:
Fixed, I left it because it works either way atm since there is only 1 flag.